src/injected/tabbable-element-getter.ts

```diff
@@ -18,14 +18,31 @@ export class TabbableElementGetter {
     ) {}
 
     public get: () => TabbableElementInfo[] = () => {
-        return this.getTabbableElements(this.doc.documentElement).map((elem, index) => ({
+        return this.getRawElements().map((elem, index) => ({
             html: elem.outerHTML,
             selector: this.generateSelector(elem as HTMLElement),
             order: index,
         }));
     };
 
     public getRawElements: () => FocusableElement[] = () => {
-        return this.getTabbableElements(this.doc.documentElement);
+        const tabbableElements = this.getTabbableElements(this.doc.documentElement);
+        const filteredElements = this.filterHiddenElements(tabbableElements);
+        return filteredElements;
+    };
+
+    private filterHiddenElements: (elements: FocusableElement[]) => FocusableElement[] = (
+        elements: FocusableElement[],
+    ) => {
+        const hiddenSelectors = ['div[aria-hidden="true"]', 'div[aria-modal="true"]'];
+        hiddenSelectors.forEach(selector => {
+            const hiddenElements = this.doc.querySelectorAll<FocusableElement>(
```

Comment:
I looked into this briefly and seems like there's a native way (node.Contains) to check whether an element is a child of another (and will also evaluate to true on itself).

So we can simplify a bit by querying the hidden selectors and then the filter method can use hiddenNode.contains(element).

I suspect that will likely faster/better since it's a browser API.